arrow-fx-reactor/src/main/kotlin/arrow/fx/reactor/extensions/common.kt

```diff
@@ -0,0 +1,7 @@
+package arrow.fx.reactor.extensions
+
+import arrow.core.Tuple2
+import java.util.function.BiFunction
+
+fun <A, B> tupled(): BiFunction<A, B, Tuple2<A, B>> =
```

Comment:
Scope it a bit so it doesn't appear on the global namespace. Internal should suffice IMO.